packages/rum-core/src/domain/rumSession.ts

```diff
@@ -26,7 +32,8 @@ export function startRumSession(configuration: Configuration, lifeCycle: LifeCyc
 
   return {
     getId: session.getId,
-    isTracked: () => session.getId() !== undefined && isTracked(session.getTrackingType()),
+    getPlan: () => getSessionPlan(session),
+    isTracked: () => getSessionPlan(session) !== undefined,
```

Comment:
nit: extracting a `isTracked` method directly could:
- be clearer here and in getSessionPlan method
- avoid to run future getSessionPlan code when only interested by isTracked info

Reply:
That was my initial thought, but then I realized we already have an `isTracked` function and I couldn't think of a good name to differentiate the two :D Let me try something.